### PR TITLE
Interface with new trycp

### PR DESCRIPTION
**Pull request by timotree3**
Depends on #57

### CODE REVIEW

**Comment by timotree3** [comment]
I believe this is complete and will be ready to review after #57 is merged.

**Comment by timotree3** [comment]
This PR updates tryorama to be able to start up and shutdown players in a remote trycp instance and adds testing for that functionality.

**Review by zippy — APPROVED** [review]
Looks great.  I tested and I see that trycp's it's up and running.  This can actually be merged as is, with the fixes to come in the next PR.

Note: I'm not totally sure about separating s.players into two functions.  I think we probably want them merged into one where the configuration can allow for both.  But we can do that as a separate step if that turns out to be needed.

File: src/conductor.ts [hunk 1]
```diff
@@ -57,14 +66,15 @@ export class Conductor {
...
     this._isInitialized = false
-    this._rawConfig = rawConfig
     this._wsClosePromise = Promise.resolve()
     this._onActivity = onActivity
   }
 
   initialize = async () => {
     this._onActivity()
-    await this._connectInterfaces()
+    if (this._adminInterfacePort !== undefined) {
```
[the remark this answers is not in the document]
Comment (timotree3) [reply src/conductor.ts:75]: tryorama never needs to know the adminInterfacePort that trycp chooses, so I don't expect this code to change

**Review by zippy — APPROVED** [review]
Looks good!

**Pull request merged**